services/management/ia/templates_service.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/percona-platform/saas/pkg/alert"
+	saas "github.com/percona-platform/saas/pkg/alert"
```

Comment:
duplicate?

Reply:
I did not do these changes, my changes are related only to `parser`. I'll recreate a PR to not confuse anybody

Reply:
That's a result of incorrect merging and conflicts resolving

Reply:
Thanks. Creating new PR right now